scalding-core/src/main/scala/com/twitter/scalding/RichPipe.scala

```diff
@@ -52,6 +53,38 @@ object RichPipe extends java.io.Serializable {
     }
     p
   }
+
+  // A pipe can have more than one description when merged together, so we store them delimited witg 254.toChar.
+  def encodePipeDescriptions(descriptions: Seq[String]): String = {
```

Comment:
make private as much as possible to make it easier to preserve binary compatibility.